packages/design-system/src/rwa/components/modal/modal-manager.tsx

```diff
@@ -0,0 +1,70 @@
+import { ModalPropsMapping, modals, ModalType } from '@/rwa';
```

Comment:
In the future, we should think about Connect providing the editor component a way to show a modal. It would keep the UX more consistent as modals can de done in a million different ways.